themes/bootstrap5/templates/Recommend/SideFacets/single-facet.phtml

```diff
@@ -57,18 +57,9 @@
 <?php $badgeExclude = ob_get_clean(); ?>
 
 <?php // Render the element: ?>
-<?php if ($hasSubLinks): ?>
```

Comment:
Does `$hasSubLinks` really no longer need to exist? If so, you can probably stop defining it above.

Reply:
Yes, removed.